nhs/data/handling.py

```diff
@@ -103,4 +106,68 @@ def read_spreadsheets(
             lambda x: x[0], placeholder_matches(files, file_dir_pattern, ["key"])
         )
 
-    return {key: val for key, val in zip(keys, map(reader, files))}
+    return {key: reader(file, sheet_id) for key, file in zip(keys, files)} if extension == "xlsx" else {
+        key: reader(file) for key, file in zip(keys, files)
+    }
+
+
+@log_entry_exit(level="INFO")
+def standarise_names(df_dict: dict[str, pl.LazyFrame | None], census_metadata: pl.LazyFrame | None,
+                     identification: str, abbreviation_column_name: str,
+                     long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+    """
+    Process the column names of a polar Lazy frame dictionary to make them more readable.
+        1. Expand abbreviated names.
+        2. Converting to lower case.
+    If any parameters are None, this function returns prev_csv_pl_df without change.
+    The filename in both dictionaries are checked or filtered to be passed.
```

Comment:
Separate one-line description from the rest of the text using a new line.

```python
"""
Standardise the column names of a polar Lazy frame dictionary to make them more readable

This function will
    1. Expand abbreviated names.
    2. Converting to lower case.
 
If any parameters are None, this function returns prev_csv_pl_df without change. The filename in both dictionaries are checked or filtered to be passed.
"""
```

Elaborate on `"The filename in both dictionaries are checked or filtered to be passed."`, what is being filtered (i.e. what names should the dictionary have)? You can say something like "keys in `df_dict` must have the format 'G{int}'" or something like that